c/extras/py_ftrl.cc

```diff
@@ -38,7 +40,7 @@ static const char* doc_lambda1  = "L1 regularization parameter";
 static const char* doc_lambda2  = "L2 regularization parameter";
 static const char* doc_d        = "Number of bins to be used for the hashing trick";
 static const char* doc_n_epochs = "Number of epochs to train a model";
-static const char* doc_inter    = "Parameter that controls if feature interactions to be used or not";
+static const char* doc_inter    = "If feature interactions to be used or not";
```

Comment:
That doesn't sound grammatically correct...

Reply:
Yes, thanks. Fixed it with "Switch to enable second order feature interaction", sounds better?